guides/common/modules/proc_migrating-to-newly-deployed-satellite.adoc

```diff
@@ -0,0 +1,57 @@
+[id="migrating-to-newly-formed-{ProjectServer}]_{context}"]
```

Comment:
anchor and title do not match the file name.

Also please remove satellite from the file name.